packages/container/src/components/FieldTypePreview.tsx

```diff
@@ -101,10 +78,17 @@ export const FieldTypePreview = forwardRef<
       />
       <FieldTypeModal isModal={props.isModal}>
         <FieldTypeContainer isModal={props.isModal}>
-          <Iframe
-            {...props}
+          <Box
             ref={ref}
             key={props.uid}
+            component="iframe"
```

Comment:
A small thing, for accessibility we could add the attribute aria-label={`Field Plugin - ${nameOfFieldPlugin}`}. This is just an example.

Reply:
I added a [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#accessibility_concerns) to the iframe.